README.md

```diff
@@ -108,3 +108,8 @@ Ensure you have access to a Unix-like environment through:
 - `rails db:migrate` Setup the database, run migrations
 - `rails assets:precompile && rails s` Compile assets & start dev server (default port _3000_)
 - `bundle exec rspec --format documentation` Run the tests (using [RSpec](http://rspec.info/) framework)
+
+### OIDC Setup
+
+- [register](https://oidc.hpi.de) a new application with the callback url `http://localhost:3000/users/auth/openid_connect/callback`
```

Comment:
love that you've thought of documenting this. let's also make sure to announce this somewhere for the whole team, so people have heard about this before hitting the problem themselves :) maybe we could also generate a dummy app, and check its credentials into the repository's `.env`?

Reply:
See updated readme, the current OIDC configuration should now not require the user to set up a client themselves.